src/field/captcha/third_party_captcha.jsx

```diff
@@ -70,6 +94,11 @@ export class ReCAPTCHA extends React.Component {
 
     script.src = scriptUrl;
     script.async = true;
+    script.defer = true;
```

Comment:
Same as with auth0.js, is setting `defer` to true critical to get the functionality work ?